### ISSUE TITLE
VSecM Init Container shall be able to wait on a Kubernetes secret

### ISSUE THREAD

**Issue by v0lkan** [vmware-tanzu/secrets-manager#763]
VSecM init container should be able to talk to VSecM Safe to check and verify that a Kubernetes Secret actually exists before yielding the flow to the main container.

Right now, VSecM init container only checks for VSecM “secret”s; not Kubernetes `Secret`s. 

To trigger initialization, the secret should be 1) there and 2) populated.

**Comment by v0lkan** [comment]
There is no need.

If the workload cannot see the Secret due to a race condition, it will crash, and when it is brought up, it will see the Secret.

A k8s API executing and creating a Secret but a workload not catching it up because the Secret was too slow to generate is a rare even. And if that happens, system will eventually reconcile.

Closing it as WONTFIX.
